src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -399,26 +401,60 @@
             <RowDefinition Height="{StaticResource HamburgerHeightGridLength}"/>
             <RowDefinition/>
         </Grid.RowDefinitions>
-        <Grid.ColumnDefinitions>
-            <ColumnDefinition Width="2*"/>
-            <ColumnDefinition Width="1*"
-                              MinWidth="300"
-                              MaxWidth="420"/>
-        </Grid.ColumnDefinitions>
         <VisualStateManager.VisualStateGroups>
-            <VisualStateGroup CurrentStateChanged="OnVisualStateChanged">
-                <VisualState x:Name="ColumnsState">
+            <VisualStateGroup>
+                <VisualState>
+                    <VisualState.StateTriggers>
+                        <states:ApplicationViewModeTrigger ViewMode="DoubleLandscape"/>
+                    </VisualState.StateTriggers>
+                    <VisualState.Setters>
+                        <Setter Target="KeyGraphFeaturesControl.(Grid.Row)" Value="0"/>
+                        <Setter Target="EquationInputAreaControl.(Grid.Row)" Value="0"/>
+                        <Setter Target="GraphingNumPad.(Grid.Row)" Value="0"/>
+                        <Setter Target="KeyGraphFeaturesControl.(Grid.RowSpan)" Value="2"/>
```

Comment:
Can we add this below to the DoublePortrait state too?

![image](https://user-images.githubusercontent.com/6334170/79026590-2fe5ce80-7b3e-11ea-838c-274baba14778.png)

Reply:
Modified!

![image](https://user-images.githubusercontent.com/1226538/79030949-852adb80-7b50-11ea-8a95-f24bf3ffa844.png)